templates/common/baremetal/files/NetworkManager-mdns-hostname.yaml

```diff
@@ -0,0 +1,21 @@
+filesystem: "root"
+mode: 0755
+path: "/etc/NetworkManager/dispatcher.d/40-mdns-hostname"
+contents:
+  inline: |
+    #!/bin/bash
+    STATUS=$2
+    case "$STATUS" in
+        dhcp4-change|dhcp6-change|hostname)
+        logger -s "NM mdns-hostname triggered by ${2}."
+        set +e
+        t_hostname=$(hostname)
```

Comment:
shouldn't it be the FQDN?

Reply:
Yep, I need to update also baremetal-runtimecfg to consider mdns hostname file